### DISCUSSION TITLE
Optimization of bot comment commands

### DISCUSSION THREAD

**Discussion by purelind** [PingCAP-QE/ci#2641] [General]
When we comment `/cherry-pick-invite` on cherry-picked pr to gain modify access of PR,  If there are space characters in the comment command, the command will not take effect.

comment link https://github.com/pingcap/tiflow/pull/10131#issuecomment-1831333392
<img width="998" alt="image" src="https://github.com/PingCAP-QE/ci/assets/16618216/c281ae8c-81cf-4b88-8ca7-2500b1140aeb">

**Comment by wuhuizuo** [comment]
> When we comment `/cherry-pick-invite` on cherry-picked pr to gain modify access of PR, If there are space characters in the comment command, the command will not take effect.
> 
> comment link [pingcap/tiflow#10131 (comment)](https://github.com/pingcap/tiflow/pull/10131#issuecomment-1831333392) <img alt="image" width="998" src="https://private-user-images.githubusercontent.com/16618216/286511037-c281ae8c-81cf-4b88-8ca7-2500b1140aeb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEwOTIxNDMsIm5iZiI6MTcxMTA5MTg0MywicGF0aCI6Ii8xNjYxODIxNi8yODY1MTEwMzctYzI4MWFlOGMtODFjZi00Yjg4LThjYTctMjUwMGIxMTQwYWViLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIyVDA3MTcyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlhY2Q1ODcxZmE0NjgwNDRiYTU2MTA5NWE5NDBhMTYyNGVlMzY5MTdkZDIxYzBiZjEwMTVmMjBhYTA3YmZhNmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9iQ4HqpY1jOxliBb6V6ZR7pcpknd0FcDiTakswDoQlA">

It's finished by inner lark channel.
